ontology/yaml/resources/HVAC/entity_types/VAV.yaml

```diff
@@ -348,6 +348,11 @@ VAV_ED_RMM:
   - ED
   - RMM
 
+VAV_UNDEFINED:
```

Comment:
Note in description that this should only be used in special cases where VAV information is not available.

Reply:
added